From 770172d03a18ba0bfe3934bf143a23fe304ced6d Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Fri, 19 Jul 2019 21:41:47 +0800
Subject: [PATCH] lima: allow to use 3D textures

Currently we have a texture whitelist in lima_screen_is_format_supported
function, which only lists 1D/2D textures now.

Allow 3D textures there.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 src/gallium/drivers/lima/lima_screen.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gallium/drivers/lima/lima_screen.c b/src/gallium/drivers/lima/lima_screen.c
index 9b6335ee24c..90a69c4a465 100644
--- a/src/gallium/drivers/lima/lima_screen.c
+++ b/src/gallium/drivers/lima/lima_screen.c
@@ -285,6 +285,7 @@ lima_screen_is_format_supported(struct pipe_screen *pscreen,
    case PIPE_BUFFER:
    case PIPE_TEXTURE_1D:
    case PIPE_TEXTURE_2D:
+   case PIPE_TEXTURE_3D:
    case PIPE_TEXTURE_RECT:
    case PIPE_TEXTURE_CUBE:
       break;
-- 
2.23.0

